README.rst

```diff
@@ -45,12 +45,18 @@ for detailed information on how to use **niflexlogger-automation**.
 
 .. _tests_section:
 
-Tests
+Automated Tests
 =====
-To run the tests, you must first configure FlexLogger to load the test
-plugins that the test projects use.  To do this, copy ``tests/assets/pythonTests.config``
-to ``%public%\Documents\National Instruments\FlexLogger\Plugins\IOPlugins``, and in that file
-replace ``<path to git repo>`` with the path to the cloned repo.
+The Python API package contains a number of automated tests which should be used to
+validate API changes before submitting a pull request. If a pull request contains
```

Comment:
Should we add something to CONTRIBUTING.md? (Maybe move/copy some of the text right here?)

Should we add something to .github/PULL_REQUEST_TEMPLATE.md? Maybe a checkbox in the "What testing has been done" section, to say whether the auto tests were run?? Or maybe they'll just type it in anyway.

Reply:
It looks like CONTRIBUTING.md already says to write new tests if applicable (and to run the tests) so I think that suffices.

I do like the idea of adding a checkbox to PULL_REQUEST_TEMPLATE - I'll do that.